students/jeonghyeon/users/views.py

```diff
@@ -33,3 +33,20 @@ def post(self, request):
         except KeyError:
                 return JsonResponse({"message" : "Check Your Data"}, status = 401)
         
+class SignInView(View):
+        def post(self, request):
+            data         = json.loads(request.body)
+            try:
+                email    = data['email']
+                password = data['password']
+
+                if not User.objects.filter(email=email, password=password).exists():
+                    return JsonResponse( {"message": "INVALID_USER"}, status=401)
+                        
+                return JsonResponse({"message": "SUCCESS" }, status = 200)
+                
+                
+            except KeyError:
+                return JsonResponse( {"message": "KEYERROR"}, status=400)
```

Comment:
에러메시지 양식은 통일해주는게 좋습니다.
```suggestion
                return JsonResponse({"message": "SUCCESS" }, status = 200)
                
            except KeyError:
                return JsonResponse( {"message": "KEY_ERROR"}, status=400)
```

Reply:
수정완료